### DISCUSSION TITLE
【灌水】这就是绳网吗

### DISCUSSION THREAD

**Discussion by Maplyn** [share121/inter-knot#610] [其他]
第一次来绳网，听说这里什么委托都能发布，真的吗？
传奇绳匠法厄同也可以见到吗？委托费会不会很贵啊？

**Comment by kaijinti** [comment]
法厄同不是说被人举报后被警察抓了吗，抓的时候那个人电脑上就登着法厄同的账号，估计是真的没跑的

**Comment by lllliufeng** [comment]
楼上的，我倒是觉得法厄同不会这么轻易就被抓，最近绳网上有不是有一个升级特别快的账号吗？说不定是金蝉脱壳了
